src/components/searchConstitution.tsx

```diff
@@ -0,0 +1,33 @@
+import { IonCard, IonCardContent, IonCol, IonIcon } from "@ionic/react";
+import React from "react";
+
+interface SearchConstitutionProps {
+  provision: any;
+}
+
+export const SearchConstitution: React.FC<SearchConstitutionProps> = ({ provision }) => {
+  return (
+    <IonCard
+      class="topic-item ion-no-margin ion-margin-vertical"
+      key={provision.id}
+      routerLink={"/constitution/provision/" + provision.id}
+    >
+      <div>
+        <IonCol size="1">
+          <IonIcon
+            size="large"
+            icon={
+              "../../assets/images/Co-operative_Government_and_Intergovernmental_Relations.svg"
```

Comment:
We need to fetch the icon that is associated with the guide associated with this provision.

Reply:
Not all provisions have related guides and in a case like this, I used the shapes image.

Reply:
Or do I use the constitution icon?

Reply:
Pls use the constitution icon as a fallback.